engine/src/org/pentaho/di/repository/RepositoryImporter.java

```diff
@@ -1035,4 +1026,32 @@ public void setOverwrite( boolean overwrite ) {
   public String getVersionComment() {
     return versionComment;
   }
+
+  private void saveTransformationToRepo( TransMeta transMeta, RepositoryImportFeedbackInterface feedback )
```

Comment:
@YuryBY can we put an abstract saveToRepo( RepositoryImportFeedbackInterface feedback ) in AbstractMeta and implement it in JobMeta and TransMeta? Now that they have a common parent class, I'd like to start moving away from "if jobs do this, if trans do that" where possible